tests/common_fixtures.py

```diff
@@ -1,38 +1,66 @@
 from http import HTTPStatus
+from os import environ
 from typing import Callable, Generator
 from unittest.mock import Mock
 
 import pytest
 import requests
 import v3io.dataplane
 
+import mlrun.api.utils.singletons.db
+import mlrun.api.utils.singletons.projects_manager
+import mlrun.config
+import mlrun.utils
 from mlrun.api.db.sqldb.db import SQLDB
-from tests.conftest import init_sqldb
+from mlrun.api.db.sqldb.session import create_session, _init_engine
+from mlrun.api.initial_data import init_data
+from mlrun.api.utils.singletons.db import initialize_db
+from mlrun.config import config
+from tests.conftest import root_path, rundb_path, logs_path
 
 session_maker: Callable
 
 
+@pytest.fixture(autouse=True)
+# if we'll just call it config it may be overridden by other fixtures with the same name
+def config_do_not_override():
```

Comment:
👎 
It's very anti-python to try and prevent overriding of anything. Is this really necessary? Why would this be more prone to "accidental" overriding then anything else?

If this is some basic configuration you expect all tests to need and don't want to use the common name `config()` then maybe something a bit more mellow, like `config_test_base()` - it's informative about being a base for tests but doesn't explicitly discourage overriding